package.json

```diff
@@ -31,29 +30,35 @@
     "react-native-reanimated": "1.13.1",
     "react-native-safe-area-context": "3.1.8",
     "react-native-schedule-exact-alarm-permission": "^0.1.3",
-    "react-native-screens": "3.20.0",
+    "react-native-screens": "3.31.1",
     "react-native-splash-screen": "3.2.0",
     "react-native-svg": "12.1.0",
     "react-native-tab-view": "3.5.1",
     "react-native-vector-icons": "10.0.3",
-    "react-native-video": "5.2.1",
+    "react-native-video": "^6.1.2",
     "semver": "5.7.2"
   },
   "devDependencies": {
     "@babel/runtime": "^7.20.0",
     "@mendix/native-mobile-toolkit": "latest",
+    "@react-native/babel-preset": "^0.73.21",
+    "@react-native/eslint-config": "^0.73.2",
+    "@react-native/metro-config": "^0.73.5",
+    "@react-native/typescript-config": "^0.73.1",
+    "@types/react": "^18.3.3",
+    "babel-jest": "^29.6.3",
     "detox": "19.13.0",
+    "jest": "^29.7.0",
     "jetifier": "^1.6.3",
     "metro-react-native-babel-preset": "0.76.8",
-    "patch-package": "^8.0.0"
+    "patch-package": "^8.0.0",
+    "prettier": "^2.8.8",
+    "typescript": "^5.0.4"
   },
   "engines": {
     "node": ">=18"
   },
   "overrides": {
-    "react": "18.2.0",
```

Comment:
We should keep these overrides. We can revisit removing it after we upgrade react-native upgrade.